### ISSUE TITLE
Fix the answers for two questions in quizzes 1 and 6

### ISSUE THREAD

**Issue by shantdashjian** [stratospheric-dev/stratospheric#240]
### What can we improve?

1. In Quiz 6: In the video it says “The applications we want to deploy are called tasks in ECS language.” But the question below from the quiz wants you to select Services, rather than Tasks, to count as correct.
 
<img width="1033" alt="Screen Shot 2023-11-06 at 3 40 29 PM" src="https://github.com/stratospheric-dev/stratospheric/assets/2113918/059e743d-bddc-4e3f-8786-c6ffef75483e">

2. In Quiz 1: There is a question about the 4 different deployment philosophies. None of the answers are correct. The correct answer should be: “Manual, Scriptable, Declarative, and Programmable”. The question wants you to select one that says “Manual, Scripted, Self-Service, and Infrastructure as Code”.

### Version

Revision 1.1 (2023-04-21)

### Suggested Changes

1. The correct answer should be Tasks.
2. The correct answer should be added and is: “Manual, Scriptable, Declarative, and Programmable”.

**Comment by BjoernKW** [comment]
@shantdashjian You're right about quiz 6. We'll fix that and release a new version. As for your second point, “Manual, Scripted, Self-Service, and Infrastructure as Code” actually is synonymous with “Manual, Scriptable, Declarative, and Programmable”. The AWS documentation specifically mentions "Infrastructure as Code": https://docs.aws.amazon.com/whitepapers/latest/introduction-devops-aws/infrastructure-as-code.html

**Comment by shantdashjian** [comment]
Yes, the AWS documentation specifically mentions "Infrastructure as Code". You also mention it in the video. That is not the problem with the answer.

The problem is that the answer "Manual, Scripted, Self-Service, and Infrastructure as Code" has an issue from a learning standpoint and correctness, that makes it not fully synonymous with “Manual, Scriptable, Declarative, and Programmable”:

For the two answers to be synonymous, you are saying that "Self-Service" is synonymous with "Declarative". It is not. Sure, "declarative" is "Self-Service" but so is Manual, Scriptable, and Programmable. All the other three are also self-service. You began the course by saying that what defines cloud is that it is self-service. So to come back and just single one of the cloud deployment approaches as self-service is confusing and contradicts the fact which you stated earlier. All approaches to cloud deployment are self-service. Manual is self-service. Scriptable is self-service, and so is Programmable. To put is in other words: "self-service" is not one of the four approaches to deployment. So it is not correct to present it as such.

I struggled with the answer as a learner and I am explaining why I struggled. I hope this reply clarifies it to you.
